pkg/apis/apiserverinternal/validation/validation.go

```diff
@@ -30,10 +30,27 @@ import (
 // ValidateStorageVersion validate the storage version object.
 func ValidateStorageVersion(sv *apiserverinternal.StorageVersion) field.ErrorList {
 	var allErrs field.ErrorList
+	allErrs = append(allErrs, validateStorageVersionName(sv.Name, field.NewPath("metadata", "name"))...)
```

Comment:
the normal mechanism for this is 

```go
// ValidateStorageVersionName is a ValidateNameFunc for storage version names
func ValidateStorageVersionName(name string, prefix bool) []string {
	var allErrs []string
	idx := strings.LastIndex(name, ".")
	if idx < 0 {
		allErrs = append(allErrs, "name must be in the form of <group>.<resource>")
	} else {
		...
	}
	return allErrs
}

func ValidateStorageVersion(sv *apiserverinternal.StorageVersion) field.ErrorList {
	var allErrs field.ErrorList
	allErrs = append(allErrs, ValidateObjectMeta(&sv.ObjectMeta, false, ValidateStorageVersionName, field.NewPath("metadata"))...)
	...
}
```

Reply:
@liggitt Thanks. I updated the code to match the pattern. Please take a look